.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: 🚀 Feature Request
+about: I have a suggestion (and may want to implement it 🙂)!
+
+---
+<!--
+You have an idea how to improve the site? That's awesome!
+
+Before submitting, please have a look at the existing issues if there's already
+something related to your suggestion.
+
+We are also [working on a relaunch](https://github.com/nodejs/website-redesign/issues/)
```

Comment:
Is it worth to replace the markdown markup in the comment with a more readable plain text?